tools/scrape_release_notes.py

```
@@ -0,0 +1,477 @@
+        "--index-url", default=PGDG_RELEASE_INDEX,
+        help="release-notes index URL (override for testing)",
+    )
+    p.add_argument("--user-agent", default=DEFAULT_USER_AGENT)
+    args = p.parse_args(argv)
+
+    majors = [m.strip() for m in args.majors.split(",") if m.strip()]
+
+    # ---- Fetch release index ----
+    try:
+        index_html = _fetch(args.index_url, user_agent=args.user_agent)
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)